lib/ActivityStream.jsm

```diff
@@ -93,18 +92,6 @@ const PREFS_CONFIG = new Map([
     title: "Remove adult pages from sites, highlights, etc.",
     value: true,
   }],
-  ["migrationExpired", {
-    title: "Boolean flag that decides whether to show the migration message or not.",
-    value: false,
```

Comment:
These prefs most likely will have user set values, so they'll stick around even when we the default values. That could be useful to keep around if something else might depend on those values down the line, so I guess we shouldn't proactively clean them up yet…